egs/datasets/README.md

```diff
@@ -52,6 +53,23 @@ The official CSD dataset can be download [here](https://zenodo.org/records/47850
  ┣ README
 ```
 
+## CustomSVCDataset
+
+We support custom dataset for Singing Voice Conversion. Arrange your data in the following structure to construct your own dataset:
+
+```plaintext
+[Your Custom Dataset Path]
+ ┣ singer1
+ ┃ ┣ song1
+ ┃ ┃ ┣ utterance1.wav
+ ┃ ┃ ┣ utterance2.wav
+ ┃ ┃ ┣ ...
+ ┃ ┣ song2
+ ┃ ┣ ...
+ ┣ singer2
+ ┣ ...
+```
+
 ## KiSing
 
 The official KiSing dataset can be download [here](http://shijt.site/index.php/2021/05/16/kising-the-first-open-source-mandarin-singing-voice-synthesis-corpus/). The file structure tree is like:
```

Comment:
-> "Download the official [KiSing dataset](link). The file structure looks like below: